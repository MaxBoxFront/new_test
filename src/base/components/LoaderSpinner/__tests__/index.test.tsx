import React from 'react';
import renderer from 'react-test-renderer';

import LoaderSpinner from 'base/components/LoaderSpinner';

describe('The LoaderSpinner component', () => {
	it('should renders correctly', () => {
		const tsx = <LoaderSpinner />;
		const tree = renderer.create(tsx).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
