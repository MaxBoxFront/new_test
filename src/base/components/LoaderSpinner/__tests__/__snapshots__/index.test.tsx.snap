// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The LoaderSpinner component should renders correctly 1`] = `
<div
  className="loader_spinner"
>
  <svg
    className="spinner"
    viewBox="0 0 50 50"
  >
    <circle
      className="spinner__path"
      cx="25"
      cy="25"
      fill="none"
      r="20"
      strokeWidth="5"
    />
  </svg>
</div>
`;
